task3/task_3_A/main_3_A.js

```diff
@@ -0,0 +1,78 @@
+const downloads = [
+  {
+    id: 1,
+    title: 'Recipe',
+    status: 'Done',
+  },
+  {
+    id: 2,
+    title: 'Workouts',
+    status: 'Pending',
+  },
+  {
+    id: 3,
+    title: 'Passwords',
+    status: 'Pending',
+  },
+  {
+    id: 4,
+    title: 'To Do 2021',
+    status: 'Pending',
+  },
+  {
+    id: 5,
+    title: 'Books',
+    status: 'Failed',
+  },
+];
+
+const table = document.querySelector('.table');
+const button = document.querySelector('.button');
+
+function createTable() {
+  for (const el of downloads) {
+    const tr = document.createElement('tr');
+    const td1 = document.createElement('td');
```

Comment:
`td10040404040`, this name isn't help other developers to know what's going on here.